src/media.md

```diff
@@ -6,7 +6,12 @@ locations](files.md) section. When you add media within Anki, either by
 using the paperclip icon in the [editor](editing.md) or by pasting it into
 a field, Anki will copy it from its original location into the media
 folder. This makes it easy to back up your collection’s media or move it
-to another computer.
+to another computer. 
+
+When importing a media file, please, please try not to use filenames with 
+blank spaces. Instead of importing the file "my picture.jpg", rename it to
+"my_picture.jpg", otherwise, [problems may arise](https://forums.ankiweb.net/t/new-html-editor-problem-searching-media-files-with-blank-spaces/14857)
+when [searching](searching.md) for media files.   
```

Comment:
I'm not sure there's a need for this, cqg. Anki should be able to handle the spaces fine, and most users never use Find+Replace on media files.  Asking the user to rename all their files with spaces just so that they can avoid some confusing behaviour in a rare case seems like a bit of an overreaction.

Reply:
Actually,  it seems to be bigger problem, not only related to the Find & Replace command: a standard search in 2.1.45+...
![1](https://user-images.githubusercontent.com/76106623/142259765-1b07676f-2ae0-4967-a069-61e8fc774b2b.png)

 ...won't find any file media with blank spaces  (2):
![2](https://user-images.githubusercontent.com/76106623/142259770-c3029054-f833-40d3-bfdd-35ae622fa098.png)

Doing the same with 2.1.44...
![3](https://user-images.githubusercontent.com/76106623/142260187-fc953894-7dae-44c7-a8b1-4af2ad40fc38.png)

...works as expected. 
![4](https://user-images.githubusercontent.com/76106623/142261045-c6ece008-d2da-4706-b35d-bf394fe296df.png)


If the problem can't be fixed, an alert in the manual can be useful, otherwise the user can spend sometime trying to figure out what's the problem here (real story, haha). Maybe rewording the paragraph would work better, or do you prefer not to mention the issue at all in the manual / FAQ?

Reply:
Mentioning it is fine, but rather than begging the user to avoid spaces, I think we should say something more like this:

If your media filenames contain spaces or other special characters such as percentage signs, the way the filenames appear in the HTML editor will differ from the way the filenames appear on disk. For example, a file called `hello 100%.jpg` will appear as `hello%20100%25.jpg` in the HTML editor. Internally, Anki still uses the original filenames, so if you would like to modify the filename with Find&Replace, you would need to use the name as it appears on disk., not as it appears in the HTML editor. Exporting to a text file is another way to see the underlying representation.

Reply:
Yes, that's better. I've applied the suggested changes, adding "so if you would like to **search for the file or** modify the filename with Find&Replace" if that's ok. 

Thanks!